frontend/src/pages/index.tsx

```diff
@@ -1,8 +1,66 @@
 /* eslint-disable canonical/filename-match-exported */
 import { type NextPage } from 'next';
+import { useState } from 'react';
+import { Modal } from '@/components/Modal';
 
 const Index: NextPage = () => {
-  return <h1>Welcome to Contra!</h1>;
+  const [modalAOpen, setModalAOpen] = useState(false);
```

Comment:
on this file i was trying to make it so we were able to open a modal form within a modal, (which is not such a good UX practice), but the goal was to test couple of edge cases. I also had buttons side by side opening or closing modals.